test.ts

```diff
@@ -1,10 +1,10 @@
-import Terraformer = require("./terraformer");
+import * as Terraformer from "./index";
 
 console.assert(typeof Terraformer !== undefined);
 
 let point = new Terraformer.Primitive({
   type: "Point",
-  coordinates: [1, 2]
+  coordinates: [1, 2],
```

Comment:
you're a hero. i'll find time to pull this down and kick the tires in the next couple of days.